dataland-community-manager/src/main/kotlin/org/dataland/datalandcommunitymanager/model/dataRequest/StoredDataRequestMessageObject.kt

```diff
@@ -0,0 +1,15 @@
+package org.dataland.datalandcommunitymanager.model.dataRequest
+
+/**
+ * Contains info about a stored message object on Dataland.
+ * @param contactList  a list of strings which include all contact (mail) details
+ * @param message a string of all messages which were created during the life cycle
+ * @param lastModifiedDate the date when the data request has been modified the last time
+ */
+data class StoredDataRequestMessageObject(
+    var contactList: List<String>? = null,
+
+    val message: String? = null,
```

Comment:
If a message is specified, the email has to be non null. Otherwise reject the request

Reply:
On it

Reply:
This is done and also tested

Reply:
Sounds good, what happens in the case that the email is empty and the message is also empty? Then it would be accepted, correct?

Reply:
Yes, if the message is empty, it doesn't matter whether the contact list is trivial. The only case to check is when the user apparently wants to send a message without providing a contact. However, it is now ensured that the message history is only filled (i.e. a message object created) when the contact list is non-trivial. Hence, an empty contact list + an empty message would not lead to a generated message object and the message history would just stay empty